Marlin/Marlin_main.cpp

```diff
@@ -5375,76 +5375,138 @@ inline void gcode_M303() {
 
 #if ENABLED(PNEUMATICS)
 
+  /*
```

Comment:
Minor nitpick that I don't necessary think is worth doing in this PR though something to think about - It's kind of lame that all of this solenoid specific functions are in Marlin Main. If I were architecting the project from the ground up, these would be a prime candidate for their own C and H files. Something to think about for the future but I'd say not right now lest we get stuck in pull request purgatory. In general, I hate main file bloat for organizational purposes.